src/features/user/CompleteSignup/index.tsx

```diff
@@ -74,7 +76,7 @@ export default function CompleteSignup() {
     setSnackbarOpen(false);
   };
 
-  const [type, setAccountType] = useState('individual');
+  const [type, setAccountType] = useState('person');
```

Comment:
Why don't you just change the translation from `Individual` to `Person` and leave the rest unchanged? I think you can not change the value for the `type` state as it is also used to communicate the user type into the backend and your change probably will break the API call.